File: Data Structures & Algorithms/largest-number/submission-2.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {string}
     */
    largestNumber(nums) {
        let arr = nums.map(String);
        arr.sort((a,b)=>(b+a)-(a+b));
        let res = arr.join("");
        if(res[0]=='0')
            return "0";
        return res;
    }
}

File: Data Structures & Algorithms/assign-cookies/submission-2.py
class Solution:
    def findContentChildren(self, g: List[int], s: List[int]) -> int:
        s.sort()
        g.sort()
        c=0
        sp=0
        gp=0
        while sp<len(s) and gp<len(g):
            if g[gp]> s[sp]:
                sp+=1
            else:
                sp+=1
                gp+=1
                c+=1
        return c